web/libs/editor/src/components/BottomBar/Controls.tsx

```diff
@@ -108,6 +109,8 @@ export const Controls = controlsInjector<{ annotation: MSTAnnotation }>(
       ],
     );
 
+    if (isNonEditableDraft) return null;
```

Comment:
given the lengthy function, simply access to the value and the fact it's used only here, I'd suggest to not use a var:
```suggestion
    if (annotation.isNonEditableDraft) return null;
```